src/main/java/com/example/demo/service/CrudService.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import com.example.demo.model.User;
+
+@Service
+public interface CrudService {
```

Comment:
remove @service from interface add it on class

Reply:
This is done